pkg/fiber-middleware/oapi_validate_test.go

```diff
@@ -23,7 +23,7 @@ import (
 //go:embed test_spec.yaml
 var testSchema []byte
 
-func doGet(t *testing.T, app *fiber.App, rawURL string) (*http.Response, error) {
+func doGet(t *testing.T, app *fiber.App, rawURL string) *http.Response {
```

Comment:
These funcs should call `t.Testing()` to mark them as a testing helper utility.

`Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Helper may be called simultaneously from multiple goroutines.`

Reply:
Has this been resolved?! Most recent code doesn't look like it :o

Reply:
I don't have a `t.Testing()`?

Reply:
You mean t.Helper() right? That has been fixed